sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -405,7 +405,7 @@ def get_all_options(
     # sub-classes in the main session might be repeated. Pick last unique
     # instance of each subclass to avoid conflicts.
     subset = {}
-    parser = _BeamArgumentParser()
+    parser = _BeamArgumentParser(allow_abbrev=False)
```

Comment:
This was causing all sorts of options because --output was getting assigned to --output_executable_path (https://github.com/apache/beam/blob/f6d209ff6a3ea638f11dfb96e52f7c21292dfbc1/sdks/python/apache_beam/options/pipeline_options.py#L1693C12-L1693C34) which meant the portable runner couldn't find prism